File: 250324/배열의 특정 위치 찾기 4/find-specific-location-of-array-4.cpp
#include <iostream>
using namespace std;

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(0);
    cout.tie(0);

    int sum = 0, count = 0;
    int arr[10] = {0,};
    for(int i = 0; i < 10; i++) {
        cin >> arr[i];
        if(arr[i] == 0) {
            break;
        }
        if(arr[i] % 2 == 0) {
            sum += arr[i];
            count++;
        }
    }
    cout << count << " " << sum;
    return 0;
}

File: 250324/배열의 특정 위치 찾기 4/find-specific-location-of-array-4.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int arr[] = new int[10];
        int count = 0;
        int sum = 0;
        for(int i = 0; i < 10; i++) {
            arr[i] = sc.nextInt();
            if(arr[i] == 0) {
                break;
            }
            if(arr[i] % 2 == 0) {
                count++;
                sum += arr[i];
            }
        }
        System.out.printf("%d %d", count, sum);
        sc.close();
    }
}